R/visualization/README.md

```diff
@@ -0,0 +1,100 @@
+# Plotting Summary Tables - how to
+
+In order to generate different types of summary tables three simple steps are necessary. Please download and place in your working directory:
+1. the basic script _scib/R/visualization/knit_table.R_, which will be used by all plotting functions;
```

Comment:
Usually use `backticks` rather than _italics_ for code but super minor so only change if you can be bothered.